kamelets/salesforce-delete-sink.kamelet.yaml

```diff
@@ -0,0 +1,107 @@
+    camel.apache.org/provider: "Apache Software Foundation"
+    camel.apache.org/kamelet.group: "Salesforce"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Salesforce Delete Sink"
+    description: "Removes an object from Salesforce."
+    required:
+      - sObjectId
+      - sObjectName
+      - clientId
+      - clientSecret
+      - userName
+      - password
+    type: object
+    properties:
+      sObjectName:
+        title: Object Name
+        description: Type of the object
+        type: string
+        example: Contact
+      sObjectId:
+        title: Object Id
+        description: Id of the object
+        type: string
+      loginUrl:
+        title: Login URL
+        description: The Salesforce instance login URL
+        type: string
+        default: https://login.salesforce.com
+      clientId:
+        title: Consumer Key
+        description: The Salesforce application consumer key
+        type: string
+        x-descriptors:
+        - urn:camel:group:credentials
+      clientSecret:
+        title: Consumer Secret
+        description: The Salesforce application consumer secret
+        type: string
+        format: password
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+        - urn:camel:group:credentials
+      userName:
+        title: Username
+        description: The Salesforce username
+        type: string
+        x-descriptors:
+        - urn:camel:group:credentials
+      password:
+        title: Password
+        description: The Salesforce user password
+        type: string
+        format: password
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+        - urn:camel:group:credentials
+  types:
+    in:
+      mediaType: application/json
```

Comment:
What does the kamelet expect as an input ? 
Looking at the kamelet definition, it looks like everything that is needed can be expressed as parameter.

Reply:
Yes, there is no need for body here. Just the id. So do I change it to send the id as a string in the body and that's it?

Reply:
Improved the documentation of what kamelets expect as input.

Reply:
Added examples. The schema (except on the delete sink) is dependent on the salesforce object type.

Reply:
In this case, it does not seems that the scheme is dependant on the object type as it is always 

```json
{ "sObjectId": "XXXXX0", "sObjectName": "Contact" }
```

Unless I missed how this kamelet works

Reply:
That's the delete one, you must specify the object name (table) and object id (key).

People that understand salesforce should know what that means...

Reply:
Where do you specify the schema? On the 
types:
  in:
??

Reply:
Done.